src/services/clients/pay/ConnectorClient.class.ts

```diff
@@ -152,6 +163,41 @@ class ConnectorClient extends BaseClient {
       },
     }
   }
+
+  private get cardTypesClient() {
+    return {
+      getAll: async () => {
```

Comment:
For the new client methods, we need to update any Pact tests to use the new connector client, ensuring integration continues to work as expected.